drivers/npmx/npmx_config.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef ZEPHYR_DRIVERS_NPMX_NPMX_CONFIG_H__
```

Comment:
It is done like in https://github.com/nrfconnect/sdk-nrf/blob/main/drivers/sensor/bh1749/bh1749.h#L10

Looking into https://github.com/nrfconnect/sdk-nrf/tree/main/drivers there is no header file with the `NRF_DRIVERS_` prefix.

Why do you want to change this?

Reply:
This driver is not part of Zephyr, so such prefix looks a bit weird. That's the only reason I added this comment. But it's only a header guard, so it can be actually anything, as long as it is unique.